Tests/L0/Maven/responseM2_HOME.json

```diff
@@ -0,0 +1,68 @@
+{
+    "exec": {
```

Comment:
This response file only needs the "exec" commands that you are making for the test that uses it. You can remove the rest. This will help make sure we don't change something incorrectly and the test will still pass unexpectedly.